x-pack/plugins/integration_assistant/server/graphs/log_type_detection/prompts.ts

```diff
@@ -22,10 +22,12 @@ Here is some context for you to reference for your task, read it carefully as yo
     `Looking at the log samples , our goal is to identify the syslog type based on the guidelines below.
 <guidelines>
 - Go through each log sample and identify the log format type.
-- If the syslog samples have header and structured body then classify it as "structured".
-- If the syslog samples have header and unstructured body then classify it as "unstructured".
-- If the syslog samples follow a csv format then classify it as "csv".
-- If you do not find the log format in any of the above categories then classify it as "unsupported".
+- If the samples have a syslog header then set "header: true" , else set "header: false". If you are unable to determine the syslog header presence then set "header: false".
+- If the syslog samples have structured body then classify it as "log_type: structured".
+- If the syslog samples have unstructured body then classify it as "log_type: unstructured".
+- If the syslog samples follow a csv format then classify it as "log_type: csv".
+- If the samples are identified as "csv" and there is a csv header then set "header: true" , else set "header: false".
+- If you do not find the log format in any of the above categories then classify it as "log_type: unsupported".
```

Comment:
do we need to add/update any documentation around what we do and do not support?

Reply:
Yeah. The documentation update is a followup task for the additional log formats being introduced. I think we should make a public doc somewhere and link it to UI.. This is a change we wait for UX to get along.
@ebeahan WDYT?